watcher/watcher.go

```diff
@@ -147,23 +147,30 @@ func (w *INotifyWatcher) Run(ctx context.Context) error {
 				return oops.Wrapf(err, "filepath.Rel: %s, %s, %s", w.directory, line[0], line[2])
 			}
 
-			eventFlag := map[string]NumericEventFlag{
-				"ATTRIB":    NumericEventFlagAttributeModified,
-				"CREATE":    NumericEventFlagCreated,
-				"DELETE":    NumericEventFlagRemoved,
-				"MOVED_TO":   NumericEventFlagMovedTo,
-				"MOVED_FROM": NumericEventFlagMovedFrom,
-				"MODIFY":    NumericEventFlagUpdated,
-			}[line[1]]
-
-			if eventFlag == NumericEventFlagInvalid {
-				return oops.Wrapf(err, "unknown inotifywait event: %s", line[1])
```

Comment:
oof, the nil error strikes again. cc/ @ccfrost

Reply:
In case it leads us somewhere some day, I started a doc with ideas for how to prevent this issue: https://paper.dropbox.com/doc/Ideas-to-fix-oops.Wrapfnil--AznVxfYaExdlddVnAkRxpLAtAg-ujc0eUPIKDInrQPlmynCy